scripts/automation/trex_control_plane/interactive/trex/astf/trex_astf_client.py

```diff
@@ -1638,3 +1638,73 @@ def profiles_line(self, line):
             self.logger.info(format_text("No profiles found with desired filter.\n", "bold", "magenta"))
 
         text_tables.print_table_with_header(table, header = 'Profile states')
+
+
+
+    ################## GTPU Clients functions ##################
+
+
+    # execute 'method' for inserting tunnel info for clients
+    def insert_tunnel_in_client_record (self, client = None):
+
+        json_attr = []
+
+        for key, value in client.items():
+            json_attr.append({'client_ip' : key, 'sip': value.sip, 'dip' : value.dip, 'teid' : value.teid, "version" :value.version, "thread_id": value.thread_id})
+
+        params = {"attr": json_attr }
+
+        print(self._transmit("add_tunnel_in_client", params))
```

Comment:
can u remove the print

Reply:
I will remove it and have to replace will return statement.